dandi/files/zarr.py

```diff
@@ -42,11 +42,47 @@
 )
 
 from .bases import LocalDirectoryAsset
-from ..validate_types import Scope, Severity, ValidationOrigin, ValidationResult
+from ..validate_types import (
+    ORIGIN_VALIDATION_DANDI_ZARR,
+    Origin,
+    OriginType,
+    Scope,
+    Severity,
+    Standard,
+    ValidationResult,
+    Validator,
+)
 
 lgr = get_logger()
 
 
+def get_zarr_format_version(data: Any) -> str:
+    """
+    Get the Zarr storage specification version from a Zarr data object
+
+    Parameters
+    ----------
+    data : zarr.core.Array or zarr.hierarchy.Group
+        The Zarr data object from which to extract the storage specification version
+
+    Returns
+    -------
+    str
+        The Zarr storage specification version,
+        https://zarr-specs.readthedocs.io/en/latest/specs.html, used in the Zarr data
+        object
+    """
+    import zarr  # Delay heavy import
```

Comment:
hm, apparently we have no single test on validation of zarrs? that's odd :-/  

filed dedicated
- https://github.com/dandi/dandi-cli/issues/1605

for that